tests/test_headful.py

```diff
@@ -0,0 +1,110 @@
+"""
+Tests relating to headful mode
+"""
+import os
+import platform
+import pytest
+import tempfile
+import shutil
+
+from pyppeteer import launch
+from syncer import sync
+
+EXTENSION_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), "assets", "simple-extension")
+
+
+class TestHeadful:
```

Comment:
This class seems to be unecessary in pytest, right?

Reply:
Some tests have class and some don't, so was not sure what is the recommended pattern. So simply chose this pattern. If this is not recommended then I can remove it.

Reply:
I referenced `test_navigation.py` as per @Mattwmaster58 which has class.

Reply:
@csreddy The reason for the classes in `test_navigation` is because there's multiple groups of different tests. In this case, the class isn't necessary for organizational purposes.

Reply:
ok, will remove it.